css/style.css

```diff
@@ -1,17 +1,24 @@
 @import url('https://fonts.googleapis.com/css2?family=Outfit:wght@300;400;600&display=swap');
-
 html {
     box-sizing: border-box;
     line-height: 1.15;
 }
-  
-*, *:before, *:after {
+
+*,
+*:before,
+*:after {
     box-sizing: border-box;
     margin: 0;
     padding: 0;
 }
 
+p {
+    margin: 0px;

```

Comment:
usually p elements comes with default styles, so I recomend just use divs

Reply:
ok in css p element. Thanks